docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -140,6 +140,31 @@ The settings that Metabase defaults to here might work for you out of the box, b
 
 Each of these input boxes needs a URI that points to the location of a SAML attribute.
 
+### Group Schema
+
+The group schema setting allows you to set Metabase groups based on an attribute of your user in your SAML provider. Please note that this may not correlate to group functionality provided by your SAML provider - you may need to create a separate attribute on your users.
+
+#### Configuring the group schema in your SAML provider
+
+First, you will need to create a user attribute that you will use to indicate which Metabase groups the user should be a part of. Different SAML providers have different ways of handling this, but you will likely need to edit your user profiles. For the rest of this example, let's say that you named your attribute metabaseGroups.
+
+Once you've created your metabaseGroups attribute, you will need to update it for each user you would like to be automatically added to a Metabase group. For each of use, we recommend using the same name for the groups you would use in Metabase.
```

Comment:
`For each of use, we recommend…`

Is this supposed to be "For each of **these**"?